.env.template

```diff
@@ -11,3 +11,8 @@ TRELLO_BOARD_ID=<change me>
 
 # Trello base URL:
 TRELLO_BASE_URL=<change me>
```

Comment:
it would be good to now remove all references to trello, like these environment variables (and those in `.env.test`)